io.github.fkinoshita.Wildcard.json

```
@@ -0,0 +1,56 @@
+{
+    "id" : "io.github.fkinoshita.Wildcard",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "wildcard",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--env=RUST_LOG=wildcard=debug"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
```

you need a cleanup rule to remove the artifacts that are build only

```suggestion
            "buildsystem": "meson",
            "cleanup": [
                "*"
            ],
```

---

Thank you!